ovirtlago/prefix.py

```diff
@@ -49,6 +50,43 @@
 log_task = functools.partial(log_utils.log_task, logger=LOGGER)
 
 
+def _create_junitxml_fullfilename(
```

Comment:
The name of the method should be changed since it can be used for any file, not just junix.xml files.

Reply:
new filename: _create_output_filename

Reply:
new function: _create_output_filename